ports/nrf/Makefile

```diff
@@ -100,8 +100,8 @@ CFLAGS += $(OPTIMIZATION_FLAGS)
 CFLAGS += $(INC) -Wall -Werror -std=gnu11 -nostdlib -fshort-enums $(BASE_CFLAGS) $(CFLAGS_MOD) $(COPT)
 
 # Undo some warnings.
-# nrfx uses undefined preprocessor variables quite casually, so we can't do warning checks for these.
-CFLAGS += -Wno-undef
+## nrfx uses undefined preprocessor variables quite casually, so we can't do warning checks for these.
+#CFLAGS += -Wno-undef
```

Comment:
Want to move this comment below to state it only applies to nrfx?

Reply:
will do